ci-operator/step-registry/ipi/conf/azure/resourcegroup/ipi-conf-azure-resourcegroup-commands.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -x
```

Comment:
I'm concerned that `set -x` is going to reveal the contents of the azure secret later on in this script. The rehearsals died before it reached the client secrets, so they definitely have not been revealed. Will the contents be exposed if this runs normally?